README.md

```diff
@@ -136,3 +136,40 @@ If you do not have Python 3.10 installed, you can use root-less podman with a Py
 ```bash
 podman run --rm --tty --volume "$(pwd):/mnt:z" --workdir /mnt docker.io/library/python:3.10 bash -c 'pip install nox ; nox -s pip-compile'
 ```
+
+## Creating release tags
+
+When a tag is created in the [`ansible/ansible`](https://github.com/ansible/ansible) repository for a release or release candidate, a corresponding tag should be created in this `ansible-documentation` repository.
+
+First, install the additional tagging dependencies as follows, creating or activating a `venv` as needed:
+
+```bash
+python3 -m venv ./venv
+source ./venv/bin/activate
+pip install -r hacking/tagger/requirements.txt
+```
+
+Next, ensure that you have both [`ansible/ansible`](https://github.com/ansible/ansible) and [`ansible/ansible-documentation`](https://github.com/ansible/ansible-documentation) repositories checked out. The tool assumes that both checkouts are done from the same directory. You can set different paths to your checkouts with the `--docs` and `--core` options.
+
```

Comment:
As someone who's never done this, I'm trying to see if I can follow these instructions.  I'm unsure what this section means by 'checkout done from same directory'.

So for example, locally, I have `ansible-documentation` dir and `ansible` dir at the same level but different directories.  So a git checkout would be:
```
samccann:ansible$ git checkout devel
samccann:ansible-documentation$ git checkout devel
```
Does this mean I need the `--docs` and` --core `options?
`

Reply:
No, having `ansible/ansible` and `ansible/ansible-documentation` in the same directory is what the tool expects, so you wouldn't need to use the `--docs` and `--core` options. if you had them in different locations for some reason (say, you had docs in one place, but did a checkout of core in a temp location) then you'd need to specify one or the other specific paths. 

I'd love some advice on how to make that easier to understand from the documentation.